front-end/src/app/shared/models/scha-transaction.model.ts

```diff
@@ -255,3 +256,11 @@ export const ScheduleATransactionTypeLabels: LabelList = [
     'Earmark Receipt for Headquarters Account (Contribution)',
   ],
 ];
+
+export type AggregationGroup = 
```

Comment:
Do we want to make this an enum instead (similar to ScheduleATransactionGroups for example)?